homeassistant/components/tuya/humidifier.py

```diff
@@ -37,7 +37,7 @@ class TuyaHumidifierEntityDescription(HumidifierEntityDescription):
     "cs": TuyaHumidifierEntityDescription(
         key=DPCode.SWITCH,
         dpcode=(DPCode.SWITCH, DPCode.SWITCH_SPRAY),
-        humidity=DPCode.DEHUMIDITY_SET_VALUE,
+        humidity=DPCode.DEHUMIDIFY_SET_VALUE,
```

Comment:
It's possible this is the wrong change.

humidity is a level, the verb is to dehumidify.